universal-login-react/src/ui/WalletSelector/EnsNamePicker.tsx

```diff
@@ -41,12 +42,23 @@ export const EnsNamePicker = ({
   const [ensName, setEnsName] = useState('');
 
   const [suggestions, setSuggestions] = useState<Suggestions | undefined>({connections: [], creations: []});
+
+  const [suggestion, setSuggestion] = useState<Suggestion | undefined>({kind: 'None'});
+
+  const setAllSuggestions = (suggestions: Suggestions) => {
+    if (suggestions) {
+      setSuggestions(suggestions);
+      setSuggestion(getSuggestion(suggestions, actions, ensName));
```

Comment:
I think that setting state 2 times in one moment is bad idea (page will render 2 times). For suppose suggestion we can use just simple 'const' instead of state

Reply:
```
const suggestion = suggestions && getSuugestion(suggestions, actions, ensName);
useEffect(() => {
    setSuggestions(undefined);	   
    debouncedSuggestionsService.getSuggestions(ensName, setSuggestions);	   
  }, [ensName]);
```